version/src/main/java/org/jboss/as/version/ProductConfig.java

```diff
@@ -97,6 +100,19 @@ private ProductConfig(ModuleLoader loader, ProductConfProps productConfProps, Ma
         this.stabilities = EnumSet.range(maxStability, minStability);
     }
 
+    public static void getBannerFile(ProductConfig config, String home) {
+        if (home != null) {
+            Path bannerPath = Paths.get(home, "bin", "banner.txt");
```

Comment:
IMO this should be read as a classloader resource from the org.jboss.as.version module. I don't want to start putting a file in bin in every server we provision that's only meant for an experimental feature[1]. Having it in bin also makes the file a kind of API. That might be intentional, but the WFCORE-4917 description doesn't say anything about that being the intent.

[1] WFLY-19021 will provide a hook that can be used to disable provisioning it, but users would need to go out of their way to be aware of and take advantage of that.

Reply:
Fixed, updating the Jira accordingly